Modules/Devices/src/Devices.Application/Clients/Queries/GetClient/Handler.cs

```diff
@@ -8,18 +7,28 @@
 namespace Backbone.Modules.Devices.Application.Clients.Queries.GetClient;
 public class Handler : IRequestHandler<GetClientQuery, ClientDTO>
 {
-    private readonly IMapper _mapper;
     private readonly IOAuthClientsRepository _oAuthClientsRepository;
+    private readonly IIdentitiesRepository _identitiesRepository;
 
-    public Handler(IMapper mapper, IOAuthClientsRepository oAuthClientsRepository)
+    public Handler(IOAuthClientsRepository oAuthClientsRepository, IIdentitiesRepository identitiesRepository)
     {
         _oAuthClientsRepository = oAuthClientsRepository;
-        _mapper = mapper;
+        _identitiesRepository = identitiesRepository;
     }
     public async Task<ClientDTO> Handle(GetClientQuery request, CancellationToken cancellationToken)
     {
         var client = await _oAuthClientsRepository.Find(request.Id, cancellationToken) ?? throw new NotFoundException(nameof(OAuthClient));
-        var clientDTO = _mapper.Map<ClientDTO>(client);
+        var numberOfIdentities = await _identitiesRepository.CountByClientId(client.ClientId, cancellationToken);
+
+        var clientDTO = new ClientDTO
+        {
+            ClientId = client.ClientId,
+            DisplayName = client.DisplayName,
+            DefaultTier = client.DefaultTier,
+            CreatedAt = client.CreatedAt,
+            NumberOfIdentities = numberOfIdentities,
+            MaxIdentities = client.MaxIdentities
+        };
```

Comment:
Create a constructor that does the mapping. Just like we do it in all of the new DTOs.

Reply:
Changes made.